bot.py

```diff
@@ -2,13 +2,14 @@
 import json
 import requests
 import telebot
-
+from telebot import types
 LOKLAK_API_URL = "http://loklak.org/api/search.json?q={query}"
-
-bot = telebot.TeleBot(os.environ['LOKLAK_TELEGRAM_TOKEN'])
+#put your token in the line below
+bot = telebot.TeleBot("PUT YOUR TOKEN HERE")
```

Comment:
refactor this into a new file called `token.py` from which do

`from token import TOKEN` where
`TOKEN = '****Your long token over here***'`